internal/ethapi/api.coreth.go

```diff
@@ -49,6 +47,11 @@ type PriceOptions struct {
 // SuggestPriceOptions returns suggestions for what to display to a user for
 // current transaction fees.
 func (s *EthereumAPI) SuggestPriceOptions(ctx context.Context) (*PriceOptions, error) {
+	var (
+		minBaseFee    int64
+		bigMinBaseFee *big.Int
+	)
+
```

Comment:
Can we define these where they are used? Feels odd to define them at the top of the function and then to initialize them later.